polaris/ocean/suites/framework_pr.txt

```diff
@@ -0,0 +1,22 @@
+## pr suite
+ocean/planar/baroclinic_channel/10km/threads
+ocean/planar/baroclinic_channel/10km/decomp
+ocean/planar/baroclinic_channel/10km/restart
+ocean/planar/ice_shelf_2d/5km/z-star/default/with_restart
+ocean/planar/ice_shelf_2d/5km/z-level/default/with_restart
+ocean/planar/inertial_gravity_wave/convergence_both
+ocean/planar/internal_wave/standard/default
+ocean/planar/internal_wave/vlr/default
+# ocean/planar/manufactured_solution/convergence_both
+ocean/single_column/cvmix
+ocean/single_column/ideal_age
+ocean/spherical/icos/cosine_bell/decomp
+ocean/spherical/icos/cosine_bell/restart
+
+## viz
+ocean/planar/ice_shelf_2d/5km/z-star/default/with_viz
```

Comment:
Does this have an edge visualization plot? If not, I think baroclinic channel with viz does

Reply:
Either way, I think the `ocean/planar/baroclinic_channel/10km/default` is a good one to add.  I had forgotten that it has viz (the `baroclinic_channel` tests in the `pr` suite don't).

Reply:
Added!